x-pack/plugins/uptime/public/apps/locators/overview.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { uptimeOverviewLocatorID } from '../../../../observability/public';
+import { OVERVIEW_ROUTE } from '../../../common/constants';
+
+const formatSearchKey = (key: string, value: string) => `${key}: "${value}"`;
+
+async function navigate({ ip, hostname }: { ip?: string; hostname?: string }) {
```

Comment:
do we want to extend it bit more? perhaps add `dateRange`, `monitorId`, `journeyId` as params, I think it would be useful for future.

Reply:
Yes I'd like to definitely add more functionality to it, but it's for a bug fix so trying to keep the footprint minimal. I will create a follow-up ER.

Reply:
ER: https://github.com/elastic/kibana/issues/123073.